pom.xml

```diff
@@ -911,53 +899,7 @@
         <version>${javax.transaction.api.version}</version>
         <scope>provided</scope>
       </dependency>
-      <!-- maven deps -->
```

Comment:
we use it directly and having this here in dptMngt will insure coherence between modules

Reply:
part of documentation here https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management 
```
The dependency management section is a mechanism for centralizing dependency information. 
When you have a set of projects that inherit from a common parent, 
it's possible to put all information about the dependency in the common POM and have simpler references 
to the artifacts in the child POMs.
```